test_no_std/Cargo.toml

```diff
@@ -27,12 +27,12 @@ features = ["disable_oom", "disable_panic_handler", "disable_allocator", "sgx"]
 
 [dependencies.sp-application-crypto]
 git = "https://github.com/paritytech/substrate.git"
-version = "3.0.0"
+tag = "monthly-2021-09+1"
 default-features = false
 features = ["full_crypto"]
 
 [dependencies.sp-core]
 git = "https://github.com/paritytech/substrate.git"
-version = "3.0.0"
+tag = "monthly-2021-09+1"
```

Comment:
point all of these to branch=master and then cargo update to the september+1 SHA

Reply:
september+1 SHA seems to cause some problems for GitHub actions, locally it compiles if the [tag commit 20a9bbb](https://github.com/paritytech/substrate/commit/20a9bbb1fe47fcd62fcd64b2fa32456b4f434aaf) is selected, but GA does not find it - which makes somewhate sense as the commit [20a9bbb](https://github.com/paritytech/substrate/commit/20a9bbb1fe47fcd62fcd64b2fa32456b4f434aaf) is not in the master branch. 
 
If I instead take the [master commit dd0fd0e](https://github.com/paritytech/substrate/pull/9676/commits/dd0fd0e4b95b32d03e1756c6217b16d71a92a0e3) , it works also in GA.

Can we agree to use [dd0fd0e](https://github.com/paritytech/substrate/pull/9676/commits/dd0fd0e4b95b32d03e1756c6217b16d71a92a0e3) instead of the tag SHA?

Or do you know a better solution to my problem?

Additionally please note, that currenlty a "cargo update -p sp-std (or any substrate library) --precise dd0fd0e4b95b32d03e1756c6217b16d71a92a0e3" does not work for crates that import substrate library frame-metadata, as it has been removed from it's original position somewhen after the specific tag commit..

Reply:
For documentation purposes, the problem GA faces:
https://github.com/Polkadex-Substrate/substrate-api-client/runs/3674383024
![grafik](https://user-images.githubusercontent.com/73821294/134347398-a8ca8204-6377-4afc-8ad3-82cd8c77dea2.png)

Reply:
Yes please use the master commit dd0fd0e , it works also in GHA.